assignment1/agent.py

```diff
@@ -0,0 +1,88 @@
+from tavily import TavilyClient
+from openai import OpenAI
+from dotenv import load_dotenv
+
+import os
+import json
+import time
+from typing import List
+from openai.types.chat.chat_completion import ChatCompletionMessage
+
+
+from tools.websurfer import WebSurfer
+from prompt import SYSTEM_PROMPT
+
+
+class Agent:
+    def __init__(self):
+        load_dotenv()
+        self.tavily_client = TavilyClient(os.environ.get("TAVILY_API_KEY"))
+        self.openai_client = OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
+
+        self.tools = [WebSurfer()]
+        self.TIMEOUT = 180
+
+    def run(self):
+        print("Ask me anything: ")
+        query = input()
```

Comment:
```suggestion
        query = input("Ask me anything: ")
```

이렇게도 동일하게 작동하는거 알고 계셨나용? 😁

Reply:
오 몰랐어요 수정했습니다! 😄